tests/attr/layer/test_grad_cam.py

```diff
@@ -78,13 +81,13 @@ def test_simple_multi_input_conv(self):
 
     def _grad_cam_test_assert(
         self,
-        model,
-        target_layer,
-        test_input,
-        expected_activation,
-        additional_input=None,
-        attribute_to_layer_input=False,
-        relu_attributions=False,
+        model: Module,
+        target_layer: Module,
+        test_input: Union[Tensor, Tuple[Tensor, ...]],
+        expected_activation: Union[List[float], Tuple[List[float], List[float]], List[List[float]], Tuple[Tensor,...]],
```

Comment:
nit: Can you change Tuple[List[float], List[float]] to Tuple[List[float], ...] to support arbitrary length tuples?

Reply:
Thank you! I fixed it.